docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+```
+
+### CLI
+
+To interact with your Epinio installation, you've to download the [Epinio CLI binary](https://github.com/epinio/epinio/releases/latest). The binary is available for the three main Operating Systems (OS), so pick the one suited for your own OS.
+
+### Login
+
+The first task to perform after Epinio installation, is to [login](../references/commands/cli/epinio_login.md) with the binary you downloaded:
+
+```shell
+epinio login -u admin 'https://epinio.127.0.0.1.sslip.io'
+
+# Trust the certificate by pressing 'y' and 'enter'
+```
+
+:::tip
+
+If your local Kubernetes cluster restarts, you only need to login again with the command above. Epinio stays installed and the certificates are still valid.
+
+:::
+
+You can confirm that you're logged in by checking the Epinio settings:
+
+```shell
+epinio settings show
+```
+
+Alternatively, you can also open the Epinio URL, <https://epinio.127.0.0.1.sslip.io>, in your preferred browser and use the web UI.
+
+## Deploy your application with Epinio
+
+Now that the "operational" tasks are done, it's time to concentrate on the most important task: use Epinio to deploy your application.
+
+As [explained here](../explanations/detailed-push-process.md#7-stage), Epinio uses [Paketo buildpacks](https://paketo.io/) to create a container image for your application. This image is then used to create a container with your application, which will run on your local Kubernetes cluster.
+
+Epinio will also create a new `ingress route`, which will allow you to easily access your application once it's deployed.
+
+The whole process is handled by Epinio, which enables you to concentrate on your application rather than knowing how you'll be able to deploy it.
+
+Let's see first how to deploy a simple application:
+
+```shell
+# Example code: https://github.com/epinio/example-12factor
+
+# Move to the source code directory. Here is an example:
+cd /github/example-12factor
+
+# Deploy your application
+epinio push -n mysimpleapp
+```
+
+At the end of the deployment output, you have the URL to be used for checking your application:
+
+```shell
+Deploying application ...
+..............
+🕞  Creating application resources
+✔️  App is online.
+Name: mysimpleapp
+Namespace: workspace
+Builder Image:
+Routes:
+1: https://mysimpleapp.127.0.0.1.sslip.io
+```
+
+### List the applications deployed
+
+If you're working on many applications, it can be really useful to see when was the last time they were deployed and which URL you should use to check them.
+
+You can get the application's information with the following two commands:
+
+```shell
+# List all deployed applications
+epinio app list
+
+# Display the applications information
+epinio app show mysimpleapp
+```
+
+### View installation logs
+
+If your application couldn't be installed, you might want to check your installation logs or, even better, save them into a file for a better screening with text editor.
```

Comment:
> you might want to check your installation logs

"you might want to check your staging logs" might read better in context?

Reply:
```suggestion
If your application couldn't be deployed, you might want to check your staging logs or, even better, save them into a file for a better screening with text editor.
```